app/src/main/java/com/zulip/android/ZulipActivity.java

```diff
@@ -88,6 +112,9 @@ public boolean setViewValue(View arg0, Cursor arg1, int arg2) {
                 case R.id.name:
                     TextView name = (TextView) arg0;
                     name.setText(arg1.getString(arg2));
+                    //Change color in the drawer if this stream is inHomeView only.
+                    if (!Stream.getByName(app, arg1.getString(arg2)).getInHomeView())
+                        name.setTextColor(ContextCompat.getColor(ZulipActivity.this, android.R.color.tertiary_text_light));
```

Comment:
When these views get recycled, the text stays grey (so streams that aren't mute have grey labels 😞 ).  One way to fix this is to reset the text back to the default color if the Stream isInHomeView().